packages/components/src/Menu/Menu.tsx

```diff
@@ -54,7 +54,14 @@ export function Menu({ activator, items }: MenuProps) {
   }, [visible]);
 
   if (!activator) {
-    activator = <Button label="More Actions" icon="more" type="secondary" />;
+    activator = (
+      <Button
+        fullWidth={true}
```

Comment:
I have looked at this in and because its wrapped in an `inline-block` div this should be okay to do. Open to discuss.

Reply:
The other option would be to pass in the activator… I'm kind of torn. What you have works and I think is cleaner but, it also feels a bit odd…

Reply:
Have discussed with Sean, he's okay with this.